packages/open-truss/src/uqi/README.md

```diff
@@ -0,0 +1,116 @@
+# uqi, a unified query interface
+
+_pronounced: yoo-kee_
+
+## What is uqi?
+
+It is a query interface with pluggable adapters that makes it simple to query different data sources from a single interface.
+
+## Implemented
+
+### Type transformations
+
+OpenTruss will support a number of query engine backends and data sources. Each of these will have their own type system. OpenTruss will provide a way to transform these types to javascript types so that workflows can be built on top of the simpler javascript types.
+
+```typescript
+otType('mysql', 'int')
```

Comment:
`otDataType`? `otScalar`? `otScalarType`?

Reply:
otDataType is nice 👍